Dockerfile

```diff
@@ -27,9 +27,7 @@ COPY application/ /data/
 
 # Fix folder permissions
 RUN chown -R www-data:www-data \
-    console/runtime/ \
-    frontend/runtime/ \
-    frontend/web/assets/
```

Comment:
I ended up having to create these two folders when I brought this repo up (to fix an error about missing folders), so if you were simply removing these lines because the folders didn't exist, that might not be necessary now.